po/com.github.bcedu.vgrive.pot

```diff
@@ -300,7 +300,7 @@ msgid "Open"
 msgstr ""
 
 #: src/views/ViewConf.vala:200
-msgid "Are you sure tou want to permanently erase the items in the Trash?"
+msgid "Are you sure you want to permanently erase the items in the Trash?"
```

Comment:
If you change a strig from the pot file, then it must be changed in the code and in all po files. Otherwise this string will not be translated in any language. If you doesn't know how to do it, I will do it when this PR is merged

Reply:
Thanks for the tip. I will do it. :)